### PR TITLE
Badge name is now automatically assigned according to gym - resolves #140

### LINKED ISSUE #140: Add random generated name to generated badge of gym

### PR DESCRIPTION
**Pull request by matejos**

### CODE REVIEW

File: persistence/src/main/java/com/kodemon/persistence/dao/GymDao.java [hunk 1]
```diff
@@ -27,7 +27,7 @@
     /**
      * Returns a {@link List} of {@link Gym}s with similar cities to the given one.
      * <p/>
-     * Note: this method generates the following query: {@code … where x.userName like ?1}
```
Comment (mseleng) [inline persistence/src/main/java/com/kodemon/persistence/dao/GymDao.java]: :smile: :smile:

File: service/src/main/java/com/kodemon/service/implementations/GymServiceImpl.java [hunk 2]
```diff
@@ -451,6 +459,21 @@ public void delete(Gym gym) {
...
+    @Override
+    public List<Gym> findByBadgeNameLike(String badgeName) {
+        return gymDao.findByBadgeNameLike(badgeName);
+    }
+
+    @Override
+    public List<Gym> findByBadgeNameContaining(String string) {
+        return gymDao.findByBadgeNameContaining(string);
+    }
+
```
Comment (mseleng) [inline service/src/main/java/com/kodemon/service/implementations/GymServiceImpl.java]: :+1:

File: service/src/main/java/com/kodemon/service/interfaces/GymService.java [hunk 3]
```diff
@@ -45,7 +45,7 @@
     /**
      * Returns a {@link List} of {@link Gym}s with similar cities to the given one.
      * <p/>
-     * Note: this method generates the following query: {@code … where x.userName like ?1}
+     * Note: this method generates the following query: {@code … where x.city like ?1}
```
Comment (mseleng) [inline service/src/main/java/com/kodemon/service/interfaces/GymService.java]: :smile:

File: persistence/src/main/java/com/kodemon/persistence/dao/GymDao.java (hunk 1 above)
Comment (matejos) [inline persistence/src/main/java/com/kodemon/persistence/dao/GymDao.java]: Somebody is very good at copypasting.. 😄

**Pull request merged**